adminSite/client/DeployStatusPage.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react"
+import { observer } from "mobx-react"
+import { observable, runInAction } from "mobx"
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faCheckCircle } from "@fortawesome/free-solid-svg-icons/faCheckCircle"
+
+import { AdminLayout } from "./AdminLayout"
+import { AdminAppContext, AdminAppContextType } from "./AdminAppContext"
+import { Deploy } from "deploy/types"
+
+@observer
+export class DeployStatusPage extends React.Component {
+    static contextType = AdminAppContext
+    context!: AdminAppContextType
+
+    @observable deploys: Deploy[] = []
+
+    render() {
+        return (
+            <AdminLayout title="Deploys">
+                <main className="DeploysPage">
+                    <h1>Deploy status</h1>
+                    {this.deploys.length > 0 ? (
+                        <table className="DeploysTable">
+                            <thead>
+                                <tr>
+                                    <th>Status</th>
+                                    <th>Note</th>
+                                    <th>Author</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {this.deploys.map(deploy =>
+                                    deploy.changes.map((change, i) => (
+                                        <tr key={`${deploy.status}-${i}`}>
+                                            <td
+                                                className={`cell-status cell-status--${deploy.status}`}
+                                            >
+                                                {deploy.status}
+                                            </td>
+                                            <td className="cell-message">
+                                                {change.message}
+                                            </td>
+                                            <td className="cell-author">
```

Comment:
Would one of those "5 minutes agos" columns be worthwhile to add?

Reply:
Yeah – we don't record a time in the messages but it would be easy to add